README.md

```diff
@@ -12,19 +12,35 @@ Almond is part of Open Thing Platform, a research project led by
 prof. Monica Lam, from Stanford University.  You can find more
 information at <https://thingpedia.stanford.edu/about>.
 
-## To Run
+## Running almond-server
+
+The recommended way to run almond-server is through [podman](https://podman.io/), a replacement for [docker](https://docs.docker.com/install/) that allows
+the container to run as your regular user (and thus access PulseAudio from your normal session).
+
+To run, use the command:
+```bash
+podman run -p 3000:3000 --uidmap keep-id \
+    -v /dev/shm:/dev/shm \
+    -v $XDG_RUNTIME_DIR/pulse:/run/pulse \
+    -v ${XDG_CONFIG_HOME:-$HOME/.config}/almond-server:/var/lib/almond-server \
+    stanfordoval/almond-server
```

Comment:
Getting `Error: unable to pull stanfordoval/almond-server: image name provided is a short name and no search registries are defined in the registries config file.` Maybe include documentation on adding search registries.